src/test/resources/secondary-instance-test.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
```

Comment:
It would be nice to at least structure this like a minimal XForm, something like this:

```
<?xml version="1.0"?>
<h:html>
    <h:head>
        <h:title>Test</h:title>
        <model>
            <instance>
                <data id="test" />
            </instance>
            <instance id="esi-id-1" src="jr://file-csv/no-file-1.csv" />
            <instance id="esi-id-2" src="jr://file-csv/no-file-2.csv" />
            <instance id="esi-id-3" src="jr://file-csv/no-file-3.csv" />
        </model>
    </h:head>
    <h:body>
        <select1 ref="/select-esi-1">
          <itemset nodeset="instance('esi-id-1')/root/element">
        </select1>
        <select1 ref="/select-esi-2">
          <itemset nodeset="instance('esi-id-2')/root/element">
        </select1>
        <select ref="/select-esi-3">
          <itemset nodeset="instance('esi-id-2')/root/element">
        </select>
        <input ref="/input-pulldata-1">
            <label>
                <output value="pulldata('esi-id-1', 'element')"/>
            </label>
        </input>
        <input ref="/input-pulldata-3">
            <label>
                <output value="pulldata('esi-id-3', 'element')"/>
            </label>
        </input>
    </h:body>
</h:html>
```